contracts/modules/LicenseController.sol

```diff
@@ -20,27 +20,19 @@ contract LicenseController is
         _component = ComponentController(_getContractAddress("Component"));
     }
 
+    // ensures that calling component (productAddress) is a product
```

Comment:
this description does not match with the code inside the method.

Reply:
inside the function _getProduct() is called. if productAddress does not belong to a registered product the function reverts. where do you see a mismatch?